dandischema/models.py

```diff
@@ -190,13 +190,13 @@ class Config:
         @staticmethod
         def schema_extra(schema: Dict[str, Any], model) -> None:
             if schema["title"] == "PropertyValue":
-                schema["required"] = list(
-                    set(["value"]).union(schema.get("required", []))
+                schema["required"] = sorted(
+                    list(set(["value"]).union(schema.get("required", [])))
```

Comment:
There's no need to cast to a list before calling `sorted()`.  `sorted()` can take any iterable as input.